src/components/KTooltip/KTooltip.vue

```diff
@@ -79,6 +83,10 @@ const props = defineProps({
     type: String,
     default: '',
   },
+  tooltipId: {
+    type: String,
+    default: uuidv4(),
```

Comment:
@portikM you can’t directly call this function inside the default prop

Reply:
It's working fine but I guess I can make it a function ruturning uuid, fixing here: https://github.com/Kong/kongponents/pull/2081

Reply:
Yup already saw that. Making it a computed property in this PR